llvm/runtimes/CMakeLists.txt

```diff
@@ -236,6 +236,7 @@ function(runtime_default_target)
   endif()
 
   set_enable_per_target_runtime_dir()
+  set(LLVM_ENABLE_RUNTIMES ${ARG_RUNTIMES})
```

Comment:
Previously, we would passthrough `LLVM_ENABLE_RUNTIMES` as a global variable, but now we always pass in `RUNTIMES` as a function argument, and then set `LLVM_ENABLE_RUNTIMES` as a local variable shadowing the global one and rely on the passthrough mechanism.

That's quite obscure and potentially error prone, I think we should stop using the passthrough machinery and instead pass `LLVM_ENABLE_RUNTIMES` as an extra argument.

Reply:
I think I tired adding it to `CMAKE_ARGS` but it was being overridden by the passthrough version like you said. Changing `PASSTHROUGH` was a much bigger chance so I tried to keep it small. Maybe I could set `LLVM_ENABLE_RUNTIMES` to `""` to make it more obvious.